primitives.rb

```diff
@@ -0,0 +1,87 @@
+#float/integer block
+@divider = "---------------------------------------------"
+@result_1 = -123.to_i
+@result_2 = '6893'.next.to_i
+@result_3 = 0.to_i-1
+@result_4 = (12.to_f/4.5).ceil.to_i
+@result_5 = ((28%6)*4.7).floor.to_i
+
+def float_integer
+	
+	puts "\n\t ***Homework for lesson4*** \n\n".upcase
+	puts "\ \ I Float/integer"
+	#question 1
+	puts "#1"
+		puts "\ \ -123 it is odd?"
+    	puts "\ \ its #{@result_1.odd?}"
+    	puts @divider
+    #question 2
+    puts "#2"
+    	
+    	puts "\ \ Next number of 6893 will be #{@result_2}"
+    	puts @divider
+    #question 3
+    puts "#3"
+    	
+    	puts "\ \ Previous number of 0 is #{@result_3}"
+    	puts @divider
+    #question 4
+    puts "#4"
+    	
+    	puts "\ \ 12 - cast it to float, then devide by 4.5 then round it up = #{@result_4} "
+    	puts @divider
+    #question 5
+    puts "#5"
+    	
+    	puts "\ \ 28 - call modulo by 6, then multiply result by 4.7 and round it down = #{@result_5}"
+    	puts @divider
+    	
+end
+float_integer
+
+#string block
+puts "\n \ \ II Strings"
+@string_input = "Brand new bicycle"
+
+def strings
+	#question 1
+    puts "#1"
+    	puts "\ \ \"remove trailing special characters\""
+		puts "\ \ #{@string_input}\r\n".delete!("\r\n")
```

Comment:
Be mindful of 'destructive' methods with `!`, it works fine in this case, but can lead to unexpected results when you do not expect them. Here's a little showcase:
![image](https://user-images.githubusercontent.com/47428140/75321555-a920af00-5879-11ea-9b27-ce24462f7ca5.png)
It will return `nil` if there won't be any characters you wanted to delete. This bited me in the ass on a couple of occasions)
Check `chomp`, `chop`, `gsub` as well